passenger-service/src/main/java/com/java/akdev/passengerservice/controller/PassengerController.java

```diff
@@ -19,14 +19,14 @@
 public class PassengerController {
 
     private final PassengerService passengerService;
-    private final PassengerRatingService passengerRatingService;
 
     @GetMapping
     public ResponseEntity<Page<PassengerReadDto>> findAll(@RequestParam Integer page,
                                                           @RequestParam Integer size,
-                                                          @RequestBody SortType sortType) {
+                                                          @RequestParam SortField sortField,
+                                                          @RequestParam Order order) {
         return ResponseEntity.status(200)
```

Comment:
Вместо строковых литералов и magic numbers следует использовать константы, чтобы эти значения можно было переиспользовать. В данном случае для статуса 200 есть отдельный метод билдера:
```suggestion
        return ResponseEntity.ok()
```